ingestion_server/test/run_test.sh

```diff
@@ -9,19 +9,19 @@ endcol="\e[0m"
 # $	./test/run_test.sh test/unit_test.py
 # ```
 if [ $# -ge 1 ]; then
-	TEST_ARG="$@"
+	TEST_ARG=( "$@" )
 else
-	TEST_ARG="test/"
+	TEST_ARG=( "test/" )
 fi
 
 PYTHONWARNINGS="ignore:Unverified HTTPS request" \
 PYTHONPATH=. \
-pytest -sx -vv --disable-pytest-warnings $TEST_ARG
+pytest -sx -vv --disable-pytest-warnings "${TEST_ARG[@]}"
 
 succeeded=$?
 if [[ $succeeded -eq 0 ]]; then
-	printf "${green}:-) All tests passed${endcol}\n"
+	printf "%s:-) All tests passed${endcol}\n" "${green}"
 else
-	printf "${red}:'( Some tests did not pass${endcol}\n"
+	printf "\n\n%s:'( Some tests did not pass${endcol}\n" "${red}"
```

Comment:
Hmm, why were these changes made? Was that part of the spellcheck? I'm now seeing different output (this should be the color green):
![image](https://user-images.githubusercontent.com/10214785/193857453-eaa24fd9-9b7d-4722-9d3b-4214b7616630.png)

Reply:
IIRC it was a ShellCheck recommendation, I'll look into it.

Reply:
@MustkimKhatik the color codes were correct, the issue was that `%s` escapes the ANSI code which makes them lose their effect. Using `%b` instead fixes the issue as seen in b9b21bd.

Reply:
> @MustkimKhatik the color codes were correct, the issue was that `%s` escapes the ANSI code which makes them lose their effect. Using `%b` instead fixes the issue as seen in [b9b21bd](https://github.com/WordPress/openverse-api/commit/b9b21bde0147b70cfad97b4d98c1b0c08284a806).

Oh alright, that must clear all the things 🙌

Reply:
That fixed it for me, thanks @dhruvkb!